packages/libs/user-datasets/src/lib/Components/Detail/UserDatasetDetail.jsx

```diff
@@ -40,12 +40,10 @@ class UserDatasetDetail extends React.Component {
     this.renderHeaderSection = this.renderHeaderSection.bind(this);
     this.renderDatasetActions = this.renderDatasetActions.bind(this);
 
-    this.renderCompatibilitySection = this.renderCompatibilitySection.bind(
-      this
-    );
-    this.getCompatibilityTableColumns = this.getCompatibilityTableColumns.bind(
-      this
-    );
+    this.renderCompatibilitySection =
+      this.renderCompatibilitySection.bind(this);
+    this.getCompatibilityTableColumns =
+      this.getCompatibilityTableColumns.bind(this);
```

Comment:
Nice! I know this is a formatting change, but I haven't thought about `.bind` in a few years 😄